### PR TITLE
[TAS-2485] 💄 Revamp signature banner

### PR DESCRIPTION
**Pull request by nwingt**
image

### CODE REVIEW

**Review by williamchong — APPROVED** [review]

**Comment by nwingt** [comment]
https://github.com/user-attachments/assets/d38180de-263a-4fdd-9a67-9b4115a59763

File: src/components/NFTBook/SignatureBanner/Mockup.vue [hunk 1]
```diff
@@ -46,24 +60,85 @@ import LottieAnimation from 'lottie-web-vue';
...
+            {
+              duration: 0.1,
+              onComplete: () => {
+                this.$refs.nameText.textContent += char;
+              },
+            }
+          );
+        });
+
+        MESSAGES[index % MESSAGES.length].split('').forEach(char => {
```
Comment (williamchong) [inline src/components/NFTBook/SignatureBanner/Mockup.vue:117]: Would be nice if we randomized the order

File: src/components/NFTBook/SignatureBanner/Mockup.vue [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+import SignatureAnimation from './signature-animation.json';
+
+const deviceFrame = require('./device-frame.png');
+
+const NAMES = ['Alice', 'Bob', 'Carol', 'David', 'Edmond'];
+const MESSAGES = [
+  '不怕，我們有彼此。',
+  'Thank you for your support.',
+  '感謝支持！',
```
Comment (williamchong) [inline src/components/NFTBook/SignatureBanner/Mockup.vue:69]:
```suggestion
  '不怕，我們有彼此。',
  'Thank you for your support.',
  '感謝支持！',
  '真理在胸筆在手\n無私無畏即自由。',
  '去留肝膽兩崑崙',
  '杜絕文青！',
  '天涼好個秋',
  'my pen is blue, \nmy friend is you'
  '見書如見人',
  '臨表涕零\n不知所言',
```

File: src/components/NFTBook/SignatureBanner/Mockup.vue [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+</template>
+
+<script>
+import LottieAnimation from 'lottie-web-vue';
+
+import SignatureAnimation from './signature-animation.json';
+
+const deviceFrame = require('./device-frame.png');
+
+const NAMES = ['Alice', 'Bob', 'Carol', 'David', 'Edmond'];
```
Comment (williamchong) [inline src/components/NFTBook/SignatureBanner/Mockup.vue:65]:
```suggestion
const NAMES = [
  'Alice',
  'Bob',
  'Carol',
  'David',
  'Edmond',
  '一心',
  '有容',
  '允行',
  '修端',
  '念慈',
];
```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/NFTBook/SignatureBanner/Mockup.vue (hunk 1 above)
Comment (nwingt) [reply src/components/NFTBook/SignatureBanner/Mockup.vue]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**